### PR TITLE
Homework topic

### PR DESCRIPTION
**Pull request by ABC-Dragon-Buster**
Re PR'd to the correct location which is content.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4791,6 +4791,33 @@ label monika_urgent:
...
     return
 
+init 5 python:
+    for key in ['homework','assignment']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_homework')
+    monika_random_topics.append('monika_homework')
+
+label monika_homework:
+      m 1a "Say, [player]..."
```
Comment (monikLover) [inline Monika After Story/game/script-topics.rpy]: Please fix all indents throughout the entire topic.
1 tab = 4 spaces.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4791,6 +4791,33 @@ label monika_urgent:
...
+            m "I also suggest finishing tasks as early as possible, as cramming them last minute can be quite problematic."
+            m 1a"I hope that could've been of some use to you."
+            m 2g "I don't want my sweetheart to be struggling in school or with work, okay?"
+            m 1b "I'm sure you'll be just fine, ahaha~"
+        "No.":
+            m 1a "I'm glad to hear that then."
+            m 1d "Some people have a hard time dealing with work and it starts to pile up gradually."
+            m 1a "So keep up the good work, [player]. Don't stress yourself out too much."
+            m "I believe in you~"
+            return
```
Comment (monikLover) [inline Monika After Story/game/script-topics.rpy]: 'return' needs to be indented correctly. Refer to other topics to see correct way.

**Comment by Winblows2** [comment]
Closing due to assumption PR has been abandoned by the submitter.

**Comment by Winblows2** [comment]
Opened due to response from submitter.

**Comment by monikLover** [comment]
Issues are still prevalent. Topic was copy/pasted and left the old broken one still in the PR. Closing.

**Pull request closed without merging**